common/static/common/templates/discussion/thread-list-item.underscore

```diff
@@ -72,7 +72,18 @@
       %>
       </span>
 
-      <span class="forum-nav-thread-comments-count <% if (unread_comments_count > 0) { %>is-unread<% } %>">
+      <% if (!neverRead && unread_comments_count > 0) { %>
+        <span class="forum-nav-thread-unread-comments-count">
+          <%-
+            StringUtils.interpolate(
+              gettext('{unread_comments_count} new'),
+              {unread_comments_count: unread_comments_count}
```

Comment:
Ack, no spaces is what's in the styleguide? I don't remember us discussing it at all, which means I picked that override on my own.... but I prefer spaces??!

Sounds like something we should revisit in the future. 👍 here though.